### PR TITLE
add clarification in ADR overview page

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**
The content in the ADR overview page now clarifies that ADRs are suitable for all audiences since they are clearly written and structured.

### CODE REVIEW

File: docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx [hunk 1]
```diff
@@ -4,14 +4,10 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 # Decisions and ADRs
 
-Documentation on important decisions, non-tech and tech-related in the form of ADRs.
+Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought they usually cover topics exclusive to the dev realm, ADRs are very clearly written and structured in a standardized that generally makes them accessible to all the members of our cross-functional team.
```
Comment (SaiSan-WMDE) [inline docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx:7]:
```suggestion
Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes them accessible to all the members of our cross-functional team.
```

File: docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx [hunk 2]
```diff
@@ -4,14 +4,10 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 # Decisions and ADRs
 
-Documentation on important decisions, non-tech and tech-related in the form of ADRs.
+Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes them accessible to all the members of our cross-functional team.
```
Comment (SaiSan-WMDE) [inline docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx:7]:
```suggestion
Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes their content accessible to all the members of our cross-functional team.
```

File: docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx [hunk 3]
```diff
@@ -4,14 +4,10 @@ import { Meta } from '@storybook/addon-docs/blocks';
 
 # Decisions and ADRs
 
-Documentation on important decisions, non-tech and tech-related in the form of ADRs.
+Crucial decisions are documented in the form of Architecture Decision Records (or ADRs). Althought ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes their content accessible to all the members of our cross-functional team.
```
Comment (raja-wmde) [inline docs/src/doc/decisions-and-ADRs/01_overview.stories.mdx:7]:
```suggestion
Crucial decisions are documented in the form of Architecture Decision Records (ADRs). Although ADRs usually cover topics exclusive to the dev realm, the clarity in which they are written and structured makes their content accessible to all the members of our cross-functional team. You can find all development and design related decisions in this folder. 
```

**Review by raja-wmde — APPROVED** [review]
Nice introduction, just added one sentence and corrected a typo.

**Comment by tzhelyazkova** [comment]
the CI is not going to pass because of the branch name 😬

**Comment by SaiSan-WMDE** [comment]
This branch needs to go due to incorrect naming

**Pull request closed without merging**